helm/krr-enforcer/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: krr-enforcer
+description: KRR enforcer - auto apply KRR recommendations
+type: application
+
+# these are set to the right value by .github/workflows/release.yaml
+# we use 0.0.1 as a placeholder for the version` because Helm wont allow `0.0.0` and we want to be able to run
```

Comment:
the comment here doesn't match the version 0.2.0 below

Reply:
removing it